test/models/description_test.rb

```diff
@@ -204,9 +204,9 @@ def test_user_sourced_description_with_unknown_user
     desc = name_descriptions(:peltigera_user_desc)
     desc.update(user_id: nil)
 
-    assert(desc.text_name.start_with?("?'s "),
+    assert(desc.text_name.start_with?("Public"),
            "text_name of user sourced Description with unknown user should " \
-           "start_with \"?'s\" ")
+           "start_with Public ")
```

Comment:
@JoeCohen Does this test express what we wanted to happen, or just what we thought would happen? 

Note that it fails on this branch because if the user is unknown, it prints "Public".

Reply:
IDK. Maybe @mo-nathan has a better idea.